### ISSUE TITLE
Retrive notification by Email

### ISSUE THREAD

**Issue by Alkasih** [driftyco/ionic-cloud#76]
Hi, is there any way to retrieve notifications based on user email. 

In the doc it is using notification_id

` /push/notifications/:notification_id`

Thank you

**Comment by piotrowski** [comment]
No, this API endpoint doesn't exist (or at least isn't documented).

**Comment by Alkasih** [comment]
Can we make it as a feature.
